agents/kathy/src/settings.rs

```diff
@@ -10,14 +10,10 @@ mod test {
     use super::*;
     use nomad_base::{get_remotes_from_env, NomadAgent};
     use nomad_test::test_utils;
-    use nomad_xyz_configuration::{
-        agent::SignerConf, ethereum::Connection, AgentSecrets, ChainConf,
-    };
+    use nomad_xyz_configuration::AgentSecrets;
 
-    #[tokio::test]
-    #[serial_test::serial]
-    async fn it_builds_settings_from_env_mixed() {
-        test_utils::run_test_with_env("../../fixtures/env.test-signer-mixed", || async move {
+    async fn test_build_from_env_file(path: &str) {
```

Comment:
Changes in kathy settings are just drying up tests

Reply:
Cool gonna remove the tests that override with defaults. Keeping the standard env.test and env.partial files though since they are base behavior we want to test e2e